static/src/javascripts/projects/commercial/modules/sticky-inlines.ts

```diff
@@ -30,6 +31,35 @@ const immersiveBufferPx = 100;
  */
 const articleBottomBufferPx = 100;
 
+/**
+ * Add a stylesheet to the document that adds height properties for a given set of class names
+ *
+ * Note these are only above on desktop and above
+ *
+ * @param heightMapping The mapping from class name to height value in pixels
+ */
+const createHeightStyles = (
+	heightMapping: Array<[string, number]>,
+): Promise<void> => {
+	const heightClasses = heightMapping.reduce(
+		(css, [name, height]) => `${css} .${name} { height: ${height}px; }`,
+		'',
+	);
+
+	const css = `
+        @media (min-width: ${breakpoints.desktop}px) {
+            ${heightClasses}
+        }
+    `;
+
+	const style = document.createElement('style');
+
+	return fastdom.mutate(() => {
+		document.head.appendChild(style);
+		style.appendChild(document.createTextNode(css));
+	});
```

Comment:
Can we can still append to `style` before we mutate the dom?

So then we just have `document.head.appendChild(style)` inside the `fastdom.mutate` call?

```suggestion
	const style = document.createElement('style');
	style.appendChild(document.createTextNode(css))

	return fastdom.mutate(() => {
		document.head.appendChild(style);
	});
```

Reply:
Ah yes, that makes more sense